modules/nodes-nodes-audit-log-basic-viewing.adoc

```diff
@@ -73,40 +70,84 @@ $ oc adm node-logs --role=master --path=kube-apiserver/
 .Example output
 [source,terminal]
 ----
-ip-10-0-140-97.ec2.internal audit-2019-04-09T14-07-27.129.log
-ip-10-0-140-97.ec2.internal audit-2019-04-09T19-18-32.542.log
-ip-10-0-140-97.ec2.internal audit.log
-ip-10-0-153-35.ec2.internal audit-2019-04-09T19-24-22.620.log
-ip-10-0-153-35.ec2.internal audit-2019-04-09T19-51-30.905.log
-ip-10-0-153-35.ec2.internal audit.log
-ip-10-0-170-165.ec2.internal audit-2019-04-09T18-37-07.511.log
-ip-10-0-170-165.ec2.internal audit-2019-04-09T19-21-14.371.log
-ip-10-0-170-165.ec2.internal audit.log
+ci-ln-m0wpfjb-f76d1-vnb5x-master-0 audit-2021-03-09T14-07-27.129.log
+ci-ln-m0wpfjb-f76d1-vnb5x-master-0 audit.log
+ci-ln-m0wpfjb-f76d1-vnb5x-master-1 audit-2021-03-09T19-24-22.620.log
+ci-ln-m0wpfjb-f76d1-vnb5x-master-1 audit.log
+ci-ln-m0wpfjb-f76d1-vnb5x-master-2 audit-2021-03-09T18-37-07.511.log
+ci-ln-m0wpfjb-f76d1-vnb5x-master-2 audit.log
 ----
 
 .. View the Kubernetes API server log for a specific master node and timestamp or view all the logs for that master:
```

Comment:
The same suggestions apply to this sentence as mentioned in my earlier comments. I.e. you could update this line to something like the following:

".. View the Kubernetes API server log for a specific master node and timestamp, or view all the logs for the node:"